qa/suites/rgw/verify/tasks/cls.yaml

```diff
@@ -1,3 +1,8 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        debug objclass: 20
```

Comment:
does this make the osd log files significantly larger? if so we may not to merge this part

Reply:
what is the point in cls tests if we cannot use logs to figure out the issues?